sky/data/storage.py

```diff
@@ -1474,8 +1474,10 @@ def batch_gsutil_cp(self,
         ]
         copy_list = '\n'.join(
             os.path.abspath(os.path.expanduser(p)) for p in source_path_list)
+        gsutil_alias, alias_gen = data_utils.get_gsutil_command()
         sync_command = (f'echo "{copy_list}" | '
-                        f'gsutil -m cp -e -n -r -I gs://{self.name}')
+                        f'{alias_gen}; {gsutil_alias} '
```

Comment:
Should the `alias_gen` go before the `echo "{copy_list}" |`? Otherwise, the list of files will not correctly passed to the `gsutil` command?